### PR TITLE
[ADD] Inventory: add create second warehouse

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR adds a new doc detailing how to create and configure a second warehouse, and add existing inventory to it.

### CODE REVIEW

**Comment by hojo-odoo** [comment]
Hey @StraubCreative,

This PR is ready for review.

Thanks,
hojo

File: content/applications/inventory_and_mrp/inventory/management/warehouses/create_a_second_warehouse.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+If a new warehouse is created that has existing inventory, this should be added to Odoo so that
+overall product counts are accurate. To add inventory to a new warehouse, start from
+:guilabel:`Inventory` and select :menuselection:`Operations --> Inventory Adjustments --> Create`,
+then fill out the inventory adjustment form as follows:
+
+- :guilabel:`Inventory Reference`: the name or code that the inventory adjustment can be referred to
+  by
+- :guilabel:`Locations`: the location or locations where the inventory is stored; include the new
+  warehouse and any locations within it that inventory will be added to
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/warehouses/create_a_second_warehouse.rst:56]: Suggestion
```suggestion
- :guilabel:`Locations`: the location(s) where the inventory is stored; include the new warehouse
  and any locations within it that inventory will be added to
```

File: content/applications/inventory_and_mrp/inventory/management/warehouses/create_a_second_warehouse.rst [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+.. image:: create_a_second_warehouse/inventory_adjustment_configuration.png
+  :align: center
+  :alt: A filled out form for an inventory adjustment.
+
+Once the form is properly configured, click on :guilabel:`Start Inventory` to be taken to the next
+page where products can be added to the inventory adjustment. Add a new product by clicking on
+:guilabel:`Create` and then fill out the product line as follows:
+
+- :guilabel:`Product`: the product being added to inventory
+- :guilabel:`Location`: the location where the product is stored; this can be the new warehouse or a
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/warehouses/create_a_second_warehouse.rst:78]: Which location: the _old_ location where the stock was previously before being shifted to the new warehouse, or the _new_ location inside the additional warehouse?

File: content/applications/inventory_and_mrp/inventory/management/warehouses/create_a_second_warehouse.rst [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  location within the warehouse
+- :guilabel:`Lot/Serial Number`: the lot that the product belongs to or the serial number used to
+  identify it
+- :guilabel:`On Hand`: the total number of the product stored in the location for which inventory is
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/warehouses/create_a_second_warehouse.rst:82]: Word choice
```suggestion
- :guilabel:`On Hand`: the total quantity of the product stored in the location for which inventory is
```

File: content/applications/inventory_and_mrp/inventory/management/warehouses/create_a_second_warehouse.rst [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+- :guilabel:`UoM`: the unit of measure used for counting the product
+
+.. image:: create_a_second_warehouse/product_line_configuration.png
+  :align: center
+  :alt: Include a line for each product being added to inventory.
+
+After adding all the products already stored in the new warehouse, click
+:guilabel:`Validate Inventory` to complete the inventory adjustment. The values in the
+:guilabel:`On Hand` column will update to reflect those in the :guilabel:`Counted` column and the
+products added will appear in the inventory of the new warehouse.
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/management/warehouses/create_a_second_warehouse.rst:96]: Can break the tag
```suggestion
After adding all the products already stored in the new warehouse, click :guilabel:`Validate
Inventory` to complete the inventory adjustment. The values in the :guilabel:`On Hand` column will
update to reflect those in the :guilabel:`Counted` column and the products added will appear in the
inventory of the new warehouse.
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @hojo-odoo 

Couple quick CRs below.

For the media file names, these [need hyphens](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#media-files) (-), not underscores (_) between the keyword values. For the IA config and NW config screenshots, think we can use more descriptive/real-life values for **Inventory Reference** and **Warehouse** name too.
e.g.
- Warehouse: San Francisco Fulfillment Center / Short Name: SF1
- Inventory Reference: SF1 Initial Inventory Count

**Comment by StraubCreative** [comment]
Last thing quick:
We'll need to change the commit message both in git and here on GH so it follows the naming convention.
Can show you tomorrow 🙂

**Comment by StraubCreative** [comment]
@StraubCreative 
Pretty sure PR is g2g, however we'll need to solve merge conflicts from #3967 first otherwise we'll run into more conflicts on FWPs.

**Review by StraubCreative — APPROVED** [review]
Hi @hojo-odoo thanks for making the edits and waiting for the other PR to merge first.

Updates on [68baa81](https://github.com/odoo/documentation/pull/3842/commits/68baa81eb36d4187fa4b55702f32a93d9a02445e): 
- renamed all images with hyphens (instead of underscores), updated RST
- CI check error (100th char break on line 43)

All checks pass ✅ 

@robodoo r+

**Review by jcs-odoo — COMMENTED** [review]
Hi @hojo-odoo and @StraubCreative !

small (and late) comment about the title:
It's usually best not to use verbs in the main title and certainly not in the RST filename unless they're very specific. Verbs such as "manage" are noisy in the toc tree as they could be written on all pages. Imagine a left menu full of "create, manage, how to". It's just harder to navigate.
About "create", it's often not only about creating something but also its configuration etc.

https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#titles-and-headings

If you plan to rename the RST filename, no need to add a redirection, as it's just been created.

Have a good day/evening :)

**Comment by StraubCreative** [comment]
> 

Ah, that makes a lot of sense thanks for the advice @jcs-odoo 🙏 
@samueljlieber @hojo-odoo we can do a quick [FIX] PR to rename this file. Should be easy merge 🙂

**Pull request closed without merging**